fmriprep/workflows/bold/base.py

```diff
@@ -259,6 +260,26 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
         run_stc = True
         multiecho = False
     else:
+        # Find associated sbref, if possible
+        entities = layout.parse_file_entities(ref_file)
+        entities['type'] = 'sbref'
+        files = layout.get(**entities)
+        refbase = os.path.basename(ref_file)
+        if files and multiecho:
+            LOGGER.warning("Single-band reference found, but not supported in "
+                           "multi-echo workflows at this time. Ignoring.")
+        elif files:
+            sbref_file = files[0].filename
+            sbbase = os.path.basename(sbref_file)
+            if len(files) > 1:
+                LOGGER.warning(
+                    "Multiple single-band reference files found for {}; using "
```

Comment:
What about merging all SBRefs in one 4D file? `gen_ref` would use that correctly, I guess. (BTW, without thinking too much I'd say I'd place the merge within `init_bold_reference_wf`)

Reply:
I think the more likely case is that a bold file without an `acq-` tag finds an sbref with and another without. I don't know that in general we want to merge those.

Reply:
Yep, we don't want to merge those. However, we should make sure we pick the right SBRef in that case or otherwise the distortion will be present in the opposite PE direction.

In general, I agree on that it is probably not worth splitting hairs by merging SBRefs 👍

Reply:
Sure. The point is to warn so that users check that it's right.

The simpler route would just be to `s/bold/sbref/` the `bold_file`, and check for existence.